src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/DeploymentOptions.cs

```diff
@@ -225,7 +225,6 @@ public DeploymentOptions()
             options.DropRoleMembersNotInSource = true;
             options.IgnoreKeywordCasing = false;
             options.IgnoreSemicolonBetweenStatements = false;
```

Comment:
anything needed on Tool Service side?

Reply:
this is the only change needed for SqlToolsService. ADS gets the default options from here, so nothing needs to be updated there.